tests/testthat/error/test-slice.txt

```diff
@@ -70,5 +70,6 @@ x The subscript has the wrong type `date`.
 i It must be logical, numeric, or character.
 
 > vec_slice(1:3, matrix(TRUE, ncol = 1))
-Error: `i` must be a one-dimensional vector but has 2 dimensions.
+Error: Must subset elements with a valid subscript vector.
+x The subscript must be a simple vector, not a matrix.
```

Comment:
Can we add the argument to the message?

```suggestion
x The subscript `1:3` must be a simple vector, not a matrix.
```

Reply:
I don't think so because `vec_slice()` is not user-oriented.

Reply:
tibble is user-oriented and sets `arg` everywhere now.

Reply:
Oh, got you -- need to set arg manually.

Reply:
Oh, you can't set the arg...

Reply:
Maybe `arg` should be quoted, than you'd do `{{`. But this should all happen lazily.

Reply:
No need for further action here, seems I can override in an error handler.